Makefile

```diff
@@ -84,10 +85,20 @@ network:
 linux:
 	docker run --rm -i --privileged --network=host --pid=host alpine nsenter -t 1 -m -u -n -i -- \
 		bash -c "ip link add dummy0 type dummy; ip addr add $(KITT_IP)/32 dev dummy0; ip link set dev dummy0 up"
+	docker run --rm -i --privileged --network=host --pid=host alpine nsenter -t 1 -m -u -n -i -- \
```

Comment:
Using nsenter like this probably won’t work on the Docker for Mac Linux VM, it is very strange.  Can you separate this into its own Linux-dns make targets, and just run the commands directly?

Reply:
i was having trouble using sudo from within the Makefile. the nsenter setup skipped that step. I'm not sure what a better alternative is atm (I'm not very familiar with Makefiles)

Reply:
Neat way to escalate privileges!  Then continue to use nsenter but move these commands into another make target.  If I’m in macOS, I can’t run these nsenter commands for systems config.

We will need os specific make entry points, make macOS-up, macOS-down, etc.